Evaluation.html

```diff
@@ -5,23 +5,22 @@
 </head>
 <body>
 	<header>
-		<ul class="pageHeader">
-			<li >
+		<ul class="header headerT">
```

Comment:
Just want to know also if is ok how I put the idea in practice. I don't know if I did already because this is the first time I have tried this and it seems a little bit odd for me